builtin/logical/database/cmd/database/main.go

```diff
@@ -0,0 +1,29 @@
+package main
```

Comment:
Database won't be able to be run as a plugin so we shouldn't offer a main file for it

Reply:
Oh right! Will remove it.